### ISSUE TITLE
[RC Feedback] Feedback on the 1.4.1rc1 Release Candidate

### ISSUE THREAD

**Issue by foosel** [OctoPrint/OctoPrint#3626]
Please provide *general* feedback on your experience with the 1.4.1rc1 Release Candidate here. An "All is working fine" is valuable feedback as well, because it tells me that people in fact are testing the RC and just not finding any problems. Thanks :)

**If you run into any obvious bugs not yet listed below the following line, please open a new ticket and follow ["How to file a bug report"](https://github.com/foosel/OctoPrint/blob/master/CONTRIBUTING.md#how-to-file-a-bug-report).**

---

### Currently known issues

  * ✔🐛 #3627 - Auto detection fails for port AUTO + set baudrate - Fix ready for 1.4.1rc2
  * ✔🐛 #3628 - German translation messes up settings - Fix ready for 1.4.1rc2
  * ✔🐛 #3629 - Logging error from tracking plugin on shutdown - Fix ready for 1.4.1rc2
  * ✔🐛 #3630 - Single file plugin install fails under Python 2 - Fix ready for 1.4.1rc2
  * ✔🐛 #3633 - High CPU while connected - Fix ready for 1.4.1rc2

**Comment by schnello** [comment]
System Klipper with TMC UART
1.4.0 > All is Fine
1.4.1 > Klipper is now unable to init the TMC via Uart.
fallback to 1.4.0 > All is fine again.

**Comment by foosel** [comment]
@schnello in any case: 

> If you run into any obvious bugs not yet listed below the following line, please open a new ticket and follow ["How to file a bug report"](https://github.com/foosel/OctoPrint/blob/master/CONTRIBUTING.md#how-to-file-a-bug-report).

;)

**Comment by gege2b** [comment]
I'm doing a print right now, and except some plugin error (with octolapse at least - I need to do some check before saying it's a bug) and the high CPU usage, everything seems fine

**Comment by jneilliii** [comment]
Cross referencing issue related to Print Time Genius and 1.4.1. I know this isn't OctoPrint's issue, but just in case someone comes across this.

https://github.com/eyal0/OctoPrint-PrintTimeGenius/issues/202

**Comment by marknn3** [comment]
Unable to connect after update to 1.4.1rc1 on Raspberry Pi 2B with OctoPi 0.17.0.
Printer is Ender 3 with Marlin 2020-07-02 firmware on SKR Mini E3 V2.0.
Reverting back to 1.4.0 restored the connection.
```
2020-07-07 20:50:09,739 - Enabling serial logging
2020-07-07 20:50:13,533 - Changing monitoring state from "Offline" to "Detecting serial connection"
2020-07-07 20:50:13,642 - Performing autodetection with 1 port/baudrate candidates: /dev/ttyACM0@115200
2020-07-07 20:50:13,655 - Connecting to port /dev/ttyACM0, baudrate 115200
2020-07-07 20:50:18,738 - Trying port /dev/ttyACM0, baudrate 115200
2020-07-07 20:50:18,770 - Changing monitoring state from "Detecting serial connection" to "Error: No more candidates to test, and no working port/baudrate combination detected."
2020-07-07 20:50:18,801 - Send: M112
2020-07-07 20:50:18,804 - Send: N1 M112*32
2020-07-07 20:50:18,807 - Send: N2 M104 T0 S0*35
2020-07-07 20:50:18,810 - Send: N3 M140 S0*102
2020-07-07 20:50:21,405 - Changing monitoring state from "Error: No more candidates to test, and no working port/baudrate combination detected." to "Offline (Error: No more candidates to test, and no working port/baudrate combination detected.)"
2020-07-07 20:50:21,439 - Connected to: None, starting monitor
2020-07-07 20:50:21,442 - Connection closed, closing down monitor
```
Note that M112 ***is*** send to the printer, but this does shutdown the printer and is then unable to respond to any commands.

**Comment by foosel** [comment]
Sounds exactly like #3627

**Comment by ogsv** [comment]
@marknn3 Can you confirm if you are able to connect when manually setting the connection parameters? If yes, it's #3627 , but if not, we might be two ppl with the same issue...

**Comment by marknn3** [comment]
@ogsv Ah I see. When I change Serial Port from AUTO to /dev/ttyACM0 then the connection succeeds! Auto Baudrate works correctly.
So indeed, it's #3627.

**Comment by foosel** [comment]
1.4.1rc2 has just been released, new feedback ticket is here: #3638

**Comment by marknn3** [comment]
The connection problem is now solved for me in 1.4.1rc2 with Auto Serial Port.
Notice that in 1.4.1rc1 a M112 was successfully sent to the printer during connect and the printer did shutdown. This might be an undesirable side effect for not able to connect.
When I switch off M112 in Settings > Serial Connection > Behaviour, the M112 is not sent but the connection still fails.
